js/main.js

```diff
@@ -1,330 +0,0 @@
-                } else {
-                createElem(Obj[key][0],Obj[key][1]+toStr,Obj[key][2]+toStr,Obj[key][3])
-                }
-        }
-    }
- } 
- // Added content to cardCreator 
- function addContentCard(Obj){
-    for (const key in Obj){
-        const id = document.getElementById(key + cardNum)
-        if(Obj.hasOwnProperty(key) && Obj[key] !== ''){
-            
-          id.innerHTML = Obj[key]
-        }
-    }
-}
- // 
-
-
-
-function addContent(Obj) {
-    for (const key in Obj){
-        const id = document.getElementById(key)
-        if(Obj.hasOwnProperty(key) && Obj[key] !== ''){
-          id.innerHTML = Obj[key]
-        }
-    }
-}
-
-//To-DO dd x instead of 0 for keys and strings for all objects and functions
-
-
-// Objects and functions for WeatherZips
-// key: html Element, ID Name, Parent Name, Class Name(Using Bootstrap)
-const bodyDivElem = {
-
-    //cardsx: ['div','cardsx','bodyDiv','container-fluid '],
-  //  cardRowx: ['div','cardRowx','cardsx','row justify-content-around 1'],
-    cardDivx: ['div','cardDivx','cardRowx','card'],
-    cardImgx: ['img','cardImgx','cardDivx','card-img-top'],
-
-    cardBodyx: ['div','cardBodyx','cardDivx','card-body'],
-    cardTitlex: ['h3','cardTitlex','cardBodyx','card-title'],
-    cardTextx: ['p','cardTextx','cardBodyx','card-text'],
-
-    cardlux: ['lu','cardlux','cardDivx','list-group list-group-flush'],
-    cardh3x: ['h3','cardh3x','cardlux','list-group-item'],
-    cardli1x: ['li','cardli1x','cardlux','list-group-item'],
-    cardli2x: ['li','cardli2x','cardlux','list-group-item'],
-    cardli3x: ['li','cardli3x','cardlux','list-group-item'],
-    
-
-    cardDivRowx: ['div','cardDivRowx','cardDivx','row card-body'],
-    cardDivCol1x: ['div','cardDivCol1x','cardDivRowx','col-6'],
-    cardBtnDelx: ['button','cardBtnDelx','cardDivCol1x','btn btn-danger btn-sm'],
-    cardDivCol2x: ['div','cardDivCol2x','cardDivRowx','col-6'],
-    cardBtnRefx: ['button','cardBtnRefx','cardDivCol2x','btn btn-light btn-sm']
-
-}
-
-// bodyDivElem Object string Content for DOM
-
-const bodyDivContent = {
-   // cardsx: '',
-   // cardRowx: '',
-    cardDivx: '',
-    cardImgx: '',
-
-    cardBodyx: '',
-    cardTitlex: 'City',
-    cardTextx: 'Description',
-
-    cardlux: '',
-    cardh3x: 'Temperature',
-    cardli1x: 'Kelvin',
-    cardli2x: 'Celcius',
-    cardli3x: 'Farenheit',
-    
-
-    cardDivRowx: '',
-    cardDivCol1x: '',
-    cardBtnDelx: 'Delete',
-    cardDivCol2x: '',
-    cardBtnRefx: 'Refresh'
-}
-
-
-
-
-  // Click Event listener
-  function BtnHello() {
-    const inputValue = document.getElementById('cardInput').value
- 
-
-    if (inputValue < 10000 || inputValue > 99951 || inputValue == NaN){
-        alert("Not A Zip Code! Please enter a Zip Code");
-    } else if(checkZip(inputValue) == true){
-        alert('pick a different zip please')
-        
-    } else {
-        getWeather(inputValue);
-    }
```

Comment:
HTML elements have some built in browser validation. Look into giving the element a pattern attribute. 

The else if should be simply `checkZip(inputValue)` since it returns true or false.